.github/ISSUE_TEMPLATE/1_bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: 🐛 Bug report
+about: Report errors and problems
+---
+
+## What happened?
+
+<!-- A clear and concise description of what the bug is. -->
+
+## How can it be recreated?
+
+<!--
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+Include screenshots here if relevant.
+-->
+
+## What did you expect to happen?
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+***
+<!-- Don't change below this line, we will fill it out as the report is investigated -->
```

Comment:
editing descriptions is tricky for concurrency and to get notifications of what has been happening on the ticket; can be used a summary but shouldn't be the primary investigation notepad

Reply:
Where's better though? It becomes harder and harder to go through the ticket comments to find out the current state etc.

Reply:
If the ticket has such a long evolution doesn't it mean the scope is too big? When it's a bug report though the comments are a trail of clues leading to the resolution. Still good to summarize them in the description.